src/types.rs

```diff
@@ -322,56 +352,56 @@ mod type_definition_tests {
             { name: my_int, type: { type: my_int } }
          */
         IslType::named("my_int", [IslConstraint::type_constraint(IslTypeRef::anonymous([IslConstraint::type_constraint(IslTypeRef::named("my_int"))]))]),
-        TypeDefinition::named("my_int", [Constraint::type_constraint(33)]) // 0-31 are built-in types which are preloaded to the type_store
+        TypeDefinition::named("my_int", [Constraint::type_constraint(34)]) // 0-32 are built-in types which are preloaded to the type_store
```

Comment:
Using numbers to refer to specific types is hard to read. Is there any way we can look these type references up by name in the tests here?

Reply:
Reference issue: https://github.com/amzn/ion-schema-rust/issues/61